src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person: {
```

Comment:
Destructure the `person` prop directly in the function argument to simplify the code.